src/alto-ui/Form/DatePicker/DatePicker.js

```diff
@@ -32,19 +32,27 @@ class DatePicker extends React.Component {
     this.state = {
       open: false,
       month: this.getDate() || DateTime.local(),
+      manualInputValue: '',
     };
 
     this.handleFocus = this.handleFocus.bind(this);
     this.handleBlur = this.handleBlur.bind(this);
     this.handleClose = this.handleClose.bind(this);
     this.handleChangeMonth = this.handleChangeMonth.bind(this);
+    this.handleChangeInput = this.handleChangeInput.bind(this);
 
     this.inputRef = React.createRef();
   }
 
-  getDate(value) {
+  getDate(value, manualInputValue) {
+    const { format } = this.props;
     const dateToParse = value || this.props.value;
     // iso or timestamp ?
+
+    if (manualInputValue) {
+      return DateTime.fromFormat(manualInputValue, format);
+    }
```

Comment:
No. remove this logic from this function. Put this directly in onBlur method or create a `parse` method (better)

Reply:
fixed